tests/test_evaluable.py

```diff
@@ -722,19 +723,19 @@ def test_loop_concatenate(self):
                      'NODES\n'
                      '%B0 = LoopConcatenate\n'
                      '├ shape[0] = 2\n'
-                     '├ start = %B1 = Take; int\n'
-                     '│ ├ %A2 = _SizesToOffsets; int; a3\n'
-                     '│ │ └ %A3 = InsertAxis; int; i2\n'
+                     '├ start = %B1 = Take; i\n'
+                     '│ ├ %A2 = _SizesToOffsets; i:a3\n'
+                     '│ │ └ %A3 = InsertAxis; i:i2\n'
                      '│ │   ├ 1\n'
                      '│ │   └ 2\n'
                      '│ └ %B4 = LoopIndex\n'
                      '│   └ length = 2\n'
-                     '├ stop = %B5 = Take; int\n'
+                     '├ stop = %B5 = Take; i\n'
                      '│ ├ %A2\n'
-                     '│ └ %B6 = Add; int\n'
+                     '│ └ %B6 = Add; i\n'
```

Comment:
@gertjanvanzwieten Is the short form desirable when `shape` is not present?

Reply:
I would say so - see comments above.